packages/console/src/components/CodeEditor/index.tsx

```diff
@@ -53,6 +53,7 @@ const CodeEditor = ({ className, language, isReadonly = false, value = '', onCha
             borderRadius: '0',
             wordBreak: 'break-all',
             fontFamily: "'Roboto Mono', monospace", // Override default font-family of <pre>
+            overflow: 'hidden',
```

Comment:
What if the content is too long (many rows)?

Reply:
Currently we don't show scrollbars inside code editor. If the content is too long, then we scroll on the outer container instead

Reply:
how about, :hover: overflow-y auto

Reply:
No... I don't think we need overflow on the `textarea` and `pre` elements. If we really want to scroll, we scroll on the outer `div` container

Reply:
Code updated. Decided to use `display: flex;` in CSS instead